static/app/views/profiling/profilingOnboarding.tsx

```diff
@@ -8,13 +9,24 @@ import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
 import {t} from 'sentry/locale';
 import {PageContent} from 'sentry/styles/organization';
 import space from 'sentry/styles/space';
-import useOrganization from 'sentry/utils/useOrganization';
+import {Organization} from 'sentry/types/organization';
+import trackAdvancedAnalyticsEvent from 'sentry/utils/analytics/trackAdvancedAnalyticsEvent';
 
-export function ProfilingOnboarding() {
-  const organization = useOrganization();
+interface Props {
```

Comment:
is this the generic `Props` name we speak of?

Reply:
That's the current styleguide... I would prefer to prefix with component name, but we should bring that up at the frontend tsc